### PR TITLE
Display restaurant suggestions in map view

### PR DESCRIPTION
**Pull request by zeroual**

### CODE REVIEW

File: frontend/src/app/home/home.component.spec.ts [hunk 1]
```diff
@@ -24,9 +25,12 @@ describe('HomeComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [HomeComponent, RestaurantComponent],
-      imports: [MatCardModule],
-      providers: [{provide: ComponentFixtureAutoDetect, useValue: true}]
+      declarations: [HomeComponent],
+      imports: [MatCardModule, MatSidenavModule, BrowserAnimationsModule],
```
Comment (RomainVernoux) [inline frontend/src/app/home/home.component.spec.ts]: These imports could be moved to a module that is shared by production code and test code (e.g. `UiModule`).

File: frontend/src/app/map-viewer/map-viewer.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import {GoogleMaps} from "./google-maps";
+import {MapMarker} from "./map-marker";
+import {BehaviorSubject} from "rxjs/index";
+
+@Component({
+  selector: 'app-map-viewer',
+  templateUrl: './map-viewer.component.html',
+  styleUrls: ['./map-viewer.component.scss']
+})
+export class MapViewerComponent implements AfterViewInit, OnDestroy {
```
Comment (RomainVernoux) [inline frontend/src/app/map-viewer/map-viewer.component.ts:11]: No unit tests for this component?

File: frontend/src/app/restaurant-suggestions-list/restaurant-suggestions-list.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RestaurantSuggestionsListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```
Comment (RomainVernoux) [inline frontend/src/app/restaurant-suggestions-list/restaurant-suggestions-list.component.spec.ts]: You should at least test that your restaurants are displayed

File: frontend/src/app/map-viewer/map-viewer.component.ts (hunk 2 above)
Comment (zeroual) [reply frontend/src/app/map-viewer/map-viewer.component.ts:11]: It's no easy to test this component.
Its integrate with Google maps and display makers .

File: frontend/src/app/restaurant-suggestions-list/restaurant-suggestions-list.component.spec.ts (hunk 3 above)
Comment (zeroual) [reply frontend/src/app/restaurant-suggestions-list/restaurant-suggestions-list.component.spec.ts]: Yeah of course . Sorry .
I didn't do it with tdd 🤣

File: frontend/src/app/home/home.component.spec.ts (hunk 1 above)
Comment (zeroual) [reply frontend/src/app/home/home.component.spec.ts]: Good idea .

File: frontend/src/app/map-viewer/map-viewer.component.ts (hunk 2 above)
Comment (RomainVernoux) [reply frontend/src/app/map-viewer/map-viewer.component.ts:11]: ok :(

**Pull request merged**